ivy/functional/frontends/torch/creation_ops.py

```diff
@@ -297,3 +301,32 @@ def zeros_like(
 ):
     ret = ivy.zeros_like(input, dtype=dtype, device=device)
     return ret
+
+
+@with_supported_dtypes({"2.0.1 and below": ("float32", "float64")}, "torch")
+@to_ivy_arrays_and_back
+@handle_out_argument
+def complex(
+    real,
+    imag,
+    *,
+    out=None,
+):
+    assert real.dtype == imag.dtype, ValueError(
+        f"Expected real and imag to have the same dtype, "
+        " but got real.dtype = {real.dtype} and imag.dtype = {imag.dtype}."
+    )
+
+    complex_dtype = ivy.complex64 if real.dtype != ivy.float64 else ivy.complex128
+    complex_array = real + imag * 1j
+    return complex_array.astype(complex_dtype)
+
+
+@with_supported_dtypes({"2.0.1 and below": ("float32", "float64")}, "torch")
+def polar(
```

Comment:
Not necessary to use `@handle_out_argument` and `@to_ivy_arrays_ and_back` in here since `complex` already outputs in `ivy.frontends.torch.Tensor` and handles `out`

Reply:
You should still use `@to_ivy_arrays_ and_back` since you're doing `angle.sin()` in it.
making sure that everything in this function is is an ivy array is better since native arrays are also expected as an input to this function and doing `angle.sin()` in that case will produce inconsistent results

Reply:
It also seems ok to me to not use `@to_ivy_arrays_ and_back` in here since there's already an implementation for both `sin` and `cos` in the torch tensor frontend (but maybe it's not the best practice (?)).

Reply:
Hi @sherry30 , could you give me a resolution on what you think would be the best course of action? given the insights I've described above

Reply:
Right, but native arrays are also expected as inputs.
So if the backend is tensorflow, tf tensors are also expected. But if you're using instance methods, it will give inconsistent results because it will call instance methods of tf tensors.
Maybe @AnnaTz can elaborate here how to proceed, I wasn't aware that `out` argument cannot be an ivy array

Reply:
Hi @dettmerramon @sherry30 
The functions of a specific frontend can receive either ivy arrays or its own frontend arrays. I think `polar` would work correctly in both cases without needing `@to_ivy_arrays_and_back`.

Reply:
tks for clarifying @AnnaTz .
@sherry30 do you have any requests left? Everything else was already implemented.

Reply:
Hi @dettmerramon, I had a talk with @sherry30 and it turns out he was right about frontend functions always needing to carry `@to_ivy_arrays_and_back`. We do need to make sure they can work for native arrays as well, so please ignore my last comment 😅